backend/apps/objects/views.py

```
@@ -335,44 +381,25 @@ def post(self, request):
+                image_url=payload["image_url"],
+                tags=payload["tags"],
                 residence=request.residence,
             )
-            if labels:
-                obj.labels.set(labels)
+            if payload["labels"]:
+                obj.labels.set(payload["labels"])
             return JsonResponse({'id': obj.id, 'detail': 'Object created successfully'}, status=201)
-        except ValueError:
-            return JsonResponse({"detail": "stock_total debe ser un entero positivo y label_ids debe ser una lista de enteros."}, status=400)
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)